README.md

```diff
@@ -30,14 +30,30 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+.forEach does not create a new array, but does return; while .map doesnt return, it does return a new array.
```

Comment:
Actually, `.map()` does utilize return values and `forEach()` doesn't return anything (`undefined`).  `forEach()` calls a provided function on each element in your array. `.map()` also calls a provided function on every element in the array.  The difference is that `map()` utilizes return values and returns a new Array of the same size.